seqr/views/utils/variant_utils.py

```diff
@@ -263,7 +270,7 @@ def _saved_variant_genes_transcripts(variants):
             TranscriptInfo.objects.filter(transcript_id__in=transcript_ids),
             nested_fields=[{'fields': ('refseqtranscript', 'refseq_id'), 'key': 'refseqId'}]
         )
-    }
+    } if transcript_ids else None
```

Comment:
The behavior is changing now such that we don't set `response['transcriptsById']` if the variants don't have genome version 38 or are mito. Why can we get rid of this?

Reply:
Because the only information in `response['transcriptsById']` is the refeseq transcript ID and whether or not a transcript is MANE. For 38 SNV/Indel calls in the hail backend, that information is now coming directly from search and we do not need to add that infornmation in a post processing step